src/extensions/default/WebPlatformDocs/main.js

```diff
@@ -69,6 +69,39 @@ define(function (require, exports, module) {
         return promiseCache[fileName];
     }
 
+    function checkProvider(hostEditor){
```

Comment:
So you're having to do this for every provider?  Is there no way to do it without touching each one?  We'll have to do it to many more than just these ones, if so.